UnityProject/Assets/Scripts/Items/HandPreparable.cs

```diff
@@ -37,7 +37,7 @@ public RightClickableResult GenerateRightClickOptions()
 		{
 			var rightClickResult = new RightClickableResult();
 			if (isPrepared) return rightClickResult;
-			rightClickResult.AddElement("Open This", Open);
+			rightClickResult.AddElement($"{openingVerb}", Open);
```

Comment:
```suggestion
			rightClickResult.AddElement(openingVerb, Open);
```
Is `openingVerb` not a string?